Gemfile.lock

```diff
@@ -0,0 +1,92 @@
+GIT
+  remote: https://github.com/openc/sqlite_magic.git
+  revision: 4df975eb4e9891de54f870077c83f63762af9bf9
```

Comment:
I didn't have a `Gemfile.lock` so this is the latest commit in that repo. I've updated the commit message.

✅

Reply:
Ok. Just to clarify, I meant _all_ the gem versions in `Gemfile.lock`. I don't think it really matters _which_ versions we ended up going with, but it would be good to just record how we ended up with them. :+1: for putting in commit message.